libraries/Microsoft.Bot.Builder.Dialogs/Choices/Channel.cs

```diff
@@ -86,11 +85,12 @@ public static bool SupportsCardActions(string channelId, int buttonCnt = 100)
         /// <returns>True if the Channel has a Message Feed, False if it does not.</returns>
         public static bool HasMessageFeed(string channelId)
         {
+            // The removed 'cortana' channel was the only channel that returned false.
+            // This channel is no longer available for bot developers and was removed from
+            // the Channels enum while addressing issues #3603, #5671
+            // Since this is publically available but not documented in the official reference docs, the method is retained.
             switch (channelId)
-            {
-                case Connector.Channels.Cortana:
-                    return false;
-
+            { 
                 default:

```

Comment:
Can we just remove the entire switch statement, and return true?

Reply:
Since the public method has the channelId parameter, removing the switch statement complains about unused variable. I can suppress the warning and remove the switch statement.